docs/extensions/module_rating.rst

```diff
@@ -0,0 +1,83 @@
+
+Tripal Module Rating System
+=============================
+
+This module rating system is meant to aid Tripal Site Administrators in choosing extension modules for their site. It is also meant to guide developers in module best practices and celebrate modules which achieve these goals.
+
+Bronze
+-------
+
+.. image:: Tripal-Bronze.png
+
+- Has a public release.
+- Should install on a Tripal site appropriate for the versions it supports.
+- Defines any custom tables or materialized views in the install file (if applicable).
+- Adds any needed controlled vocabulary terms in the install file (Tripal3)
+- Provides Installation and admin instructions README.md (or RTD)
```

Comment:
```suggestion
- Provides Installation and admin instructions README.md (or [RTD](https://tripal.readthedocs.io/en/latest/dev_guide/rtd.html))
```

Reply:
I think you can include my fix by clicking the "Approve changes" link below.... First time I've tried this.

Reply:
thats markdown formatting.  THe . correct format is:

```suggestion
- Provides Installation and admin instructions README.md (or `RTD <https://tripal.readthedocs.io/en/latest/dev_guide/rtd.html>`_)
```